src/views/Lottery/components/PastDrawsHistory/PastDrawsHistoryCard.tsx

```diff
@@ -19,15 +19,16 @@ const Wrapper = styled.div`
 `
 
 const LabelWrapper = styled.div`
-  > ${Text} {
-    font-size: 12px;
-  }
-
   margin-top: 24px;
 
   ${({ theme }) => theme.mediaQueries.sm} {
     margin-top: 0;
   }
+
+  > ${Text} {
+    font-size: 12px;
+    text-transform: uppercase;
+  }
```

Comment:
Sorry, my message was unclear. This can be set directly on the Text component.

```
<Text fontSize="12px" textTransform="uppercase">
...
```

Reply:
done